cmd/oras/internal/display/status/tty.go

```diff
@@ -143,3 +143,62 @@ func (ph *TTYPullHandler) TrackTarget(gt oras.GraphTarget) (oras.GraphTarget, St
 	ph.tracked = tracked
 	return tracked, tracked.Close, nil
 }
+
+// TTYCopyHandler handles tty status output for copy events.
+type TTYCopyHandler struct {
+	tty       *os.File
+	committed *sync.Map
+	tracked   track.GraphTarget
+}
+
+// NewTTYCopyHandler returns a new handler for copy command.
+func NewTTYCopyHandler(tty *os.File) CopyHandler {
+	return &TTYCopyHandler{
+		tty:       tty,
+		committed: &sync.Map{},
+	}
+}
+
+// StartTracking returns a tracked target from a graph target.
+func (ch *TTYCopyHandler) StartTracking(gt oras.GraphTarget) (oras.GraphTarget, error) {
+	tracked, err := track.NewTarget(gt, copyPromptCopying, copyPromptCopied, ch.tty)
+	ch.tracked = tracked
+	return tracked, err
```

Comment:
it makes more sense logically as
```suggestion
	tracked, err := track.NewTarget(gt, copyPromptCopying, copyPromptCopied, ch.tty)
	if err != nil {
		return nil, err
	}
	ch.tracked = tracked
	return tracked, nil
```

Reply:
Done. Mixed two feedbacks on this